src/cloudai/report_generator/tool/bokeh_report_tool.py

```diff
@@ -195,13 +201,26 @@ def add_log_x_linear_y_single_line_plot(
         x_min, x_max = self.find_min_max(df, x_column)
         y_min, y_max = self.find_min_max(df, y_column, sol)
 
+        # Check if x_min equals x_max - constant message size
+        if x_min == x_max:
```

Comment:
We are adding identical changes into `add_log_x_linear_y_single_line_plot` and `add_log_x_linear_y_multi_line_plot`. Is there any reason we have them as two separate functions? Looks like _single_ could be removed and we can call _multi_ with a list of a single element.